include/openssl/cryptoerr.h

```diff
@@ -14,9 +14,6 @@
 # ifdef  __cplusplus
 extern "C"
 # endif
-
-# include <openssl/symhacks.h>
-
```

Comment:
Please put that back.

Reply:
I didn't remove it; mkerr didn't add it.  I just did make update.

Reply:
I know that mkerr removed it.  mkerr isn't perfect and doesn't know that cryptoerr.h specifically needs that header, because:

https://github.com/openssl/openssl/blob/38023b87f037f4b832c236dfce2a76272be08763/include/openssl/symhacks.h#L17-L18

So, please put that back.

Reply:
So anytime someone does a 'make update' and changes cryptoerr they have to edit a generated file.  No, that's really really bad.

Reply:
Well, no one has bothered to fix that, and I guess it's because we change cryptoerr.h so seldom.

Feel free to fix that.

Reply:
#8397